Tests/otlLib/builder_test.py

```diff
@@ -1392,6 +1392,63 @@ def test_stat_infinities():
     assert struct.pack(">l", posInf) == b"\x7f\xff\xff\xff"
 
 
+class ChainContextualRulesetTest(object):
+    def test_makeRulesets(self):
+        font = ttLib.TTFont()
+        font.setGlyphOrder(["a","b","c","d","A","B","C","D","E"])
+        sb = builder.ChainContextSubstBuilder(font, None)
+        prefix = [ ["a"], ["b"] ]
```

Comment:
```suggestion
        prefix = [["a"], ["b"]]
```
and all the following lines as well